stories/core/Table.stories.tsx

```diff
@@ -39,78 +39,39 @@ export default {
     emptyTableContent: 'No data.',
   },
   argTypes: {
```

Comment:
Also add `disableFilters`

Reply:
I didn't add that because I thought it would be nice to have (as it does allow you to disable filtering in storybook)